src/App.js

```diff
@@ -1,12 +1,225 @@
-import React from 'react';
+import React, { useState } from "react";
 import './App.css';
+import FoodButton from "./components/FoodButton";
+import FormField from "./components/FormField";
+// import FormDropDown from "./components/FormDropDown";
 
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [strawberries, setStrawberries] = useState(0);
+    const [apples, setApples] = useState(0);
+    const [bananas, setBananas] = useState(0);
+    const [kiwis,setKiwis] = useState(0);
+
+    const [formState, setFormState] = useState({
+        firstname: '',
+        lastname: '',
+        age: 0,
+        zipcode: '',
+        deliveryFrequency: 'Iedere week',
+        timeOfDay: 'Overdag',
+        remark: '',
+        conditions: false,
+    });
+
+    function resetFruits () {
+        setStrawberries(0)
+        setApples(0)
+        setBananas(0)
+        setKiwis(0)
+    }
+
+    function handleChange(e) {
+        const changedFieldName = e.target.name;
+        const checkBoxValue = e.target.type === "checkbox" ? e.target.checked : e.target.value;
+
+        setFormState({
+            ...formState,
+            [changedFieldName]: checkBoxValue,
+        });
+    }
+
+    // function dropDown(optionValue, optionDescription) {
+    //     return (
+    //         <option
+    //             value={optionValue}
+    //         >{optionDescription}
+    //         </option>
+    //     )
+    // }
+
+    return (
+        <>
+            <h1>Fruitmand bezorgservice</h1>
+            <div className="app">
+                <FoodButton
+                    fruitButtonCSS="buttons"
+                    imgPlusTitle="🍓 Aardbeien"
+                    buttonDecrement="-"
+                    setFruit={setStrawberries}
+                    fruit={strawberries}
+                    buttonIncrement="+"
+                    notEnabled={strawberries === 0}
```

Comment:
Uiteindelijk zie ik best veel property waardes die altijd hetzelfde zijn, voor al deze geldt dat je dit binnen het component zelf had kunnen doen.
```suggestion
                    imgPlusTitle="🍓 Aardbeien"
                    setFruit={setStrawberries}
                    fruit={strawberries}
```